### ISSUE TITLE
FileManager support for 2021.3 snapshot build (1.17)

### ISSUE THREAD

**Issue by smmmadden** [Goldorion/File-Manager-MCreator#18]
I wanted to start testing the MCreator 2021.3 Snapshot build with my mods, but they fail with the following errors using the current filemanager.zip and build 4.0.2 patch.
Block file_exists is not supported by the selected generator!
Block read_json is not supported by the selected generator.  It will be skipped.
Block file_variable is not supported by the selected generator.  It will be skipped.
Block write_json is not supported by the selected generator.  It will be skipped.
Block read_json is not supported by the selected generator.  It will be skipped.

I'm assuming also that my last post in issue-11 isn't fixed either.  Do you have an eta on when there will be an update for 2021.3x ?

**Comment by smmmadden** [comment]
Downloaded the lastest snapshot 2021.3b47317 and see the same issues.  I'm guessing that the FileManager.zip file hasn't been updated for 1.17.1 yet?

**Comment by smmmadden** [comment]
I've upgraded to 2021.3 b49312 and tried to identify where the problem as in the plugin.  So I copied the 1.16.5 folder to 1.17.1 folder in the zip and that allowed the procedures to now save.  However, it still fails to compile, so something appears to have changed deeper in Forge or Minecraft.  Any ideas?
![image](https://user-images.githubusercontent.com/10309346/146006798-1880f274-22bf-45b3-b4a4-340dd03ba846.png)
![image](https://user-images.githubusercontent.com/10309346/146006836-6d2dfa28-f5fc-409c-abb4-632a754fabf2.png)
Thanks!
